config/rbac/servicebinding_controller_role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: controller-role
```

Comment:
@bamachrn So, this should be changed to `service-binding-operator-controller-role`, right?

Reply:
yes

Reply:
my bad no, this gets prefixed with `service-binding-operator` by kustomaization while generating the bundle